restx-apidocs-doclet/src/main/java/restx/apidocs/doclet/ApidocsDoclet.java

```diff
@@ -168,20 +167,6 @@ public static int optionLength(String option) {
         return Standard.optionLength(option);
     }
 
-    /**
-     * Processes the input options by delegating to the standard handler.
-     *
-     * _Javadoc spec requirement._
-     *
-     * @param options input option array
-     * @param errorReporter error handling
-     * @return success
-     */
-    @SuppressWarnings("UnusedDeclaration")
-    public static boolean validOptions(String[][] options, DocErrorReporter errorReporter) {
-        return Standard.validOptions(options, errorReporter);
-    }
-
```

Comment:
It seems that this will prevent the doclet from working with options on java 8 and java 9.
As stated here:
https://docs.oracle.com/javase/8/docs/jdk/api/javadoc/doclet/com/sun/javadoc/Doclet.html

This method is required by the Doclet api to accept options.

Maybe a better option would be to have a new module `restx-apidocs-doclet-11` compatible with doclet api 11, and keeping this module compatible with java < 11.

The new doclet api is documented here:
https://openjdk.java.net/groups/compiler/using-new-doclet.html

It has a brand new way to deal with options.

Reply:
Thank you for this input. I back ported it in the [issue](https://github.com/restx/restx/issues/313#issuecomment-620597520). As it implies a different solution, I'm going to close this pull request. I will work on this issue by following your guidelines.